.github/workflows/ci.yml

```diff
@@ -40,10 +40,17 @@ jobs:
       run: cabal update
       # Build macaw-base dependencies and crucible separately just so later
       # steps are less verbose and major dependency failures are separate.
+    - name: Install python dependencies
+      working-directory: ./python
+      run: |
+        if [ -f requirements.txt ]; then pip3 install -r requirements.txt; fi
     - name: Build
       run: |
         cabal build all
     - name: Cabal argo tests
       run: cabal test argo
     - name: Cabal file-echo-api tests
       run: cabal test file-echo-api
+    - name: Python Cryptol unit tests
+      working-directory: ./python/cryptol
+      run: python3 -m unittest discover
```

Comment:
One final issue: can we set this up to run as part of `cabal test`? We already run Python scripts as part of our normal test suite, and having some tests require a different command to run than the others seems like a recipe for ongoing confusion.

Reply:
https://github.com/GaloisInc/argo/pull/116/commits/1539ba21ee911cab2e5aec824b20617af65bb655 tweaks things so `cabal test python` will run the python unit test and discovery, and the success of any discovered python unit tests in the directory structure is a part of the existing Haskell test suite's success. Feels a little odd to have the haskell test framework invoking the python unit test framework... but I think it's fine for now and at least it's simple.

Reply:
Okay... but the test are failing in a way I don't see locally and specifically running mypy on the files in question doesn't produce the error either... hmmmm....